libs/langchain/langchain/llms/huggingface_hub.py

```diff
@@ -107,15 +108,17 @@ def _call(
         """
         _model_kwargs = self.model_kwargs or {}
         params = {**_model_kwargs, **kwargs}
-        response = self.client(inputs=prompt, params=params)
+        response = self.client.post(
+            json={"inputs": prompt, "params": params}, task=self.task
+        ).json()
```

Comment:
getting an error here, InferenceClient.post seems to return bytes

Reply:
@baskaryan will fix it.

Reply:
@baskaryan  I double-checked the response is a JSON. I am not so sure why you are saying it is bytes.

Reply:
@baskaryan Can you please post your error?

Reply:
?

Reply:
Don't you need to decode the output of `post` first? Also, does `bytes` have a `json()` method?
I suppose the correct thing to do here is:

```py
response = self.client.post(
   json={"inputs": prompt, "params": params}, task=self.task
)

data = json.loads(response.decode())
```